app/models/pizza.rb

```diff
@@ -0,0 +1,6 @@
+class Pizza < ApplicationRecord
+  validates :brand, presence: true
+  validates :product_name, presence: true
+  #validates :cost, :allow_nil true
```

Comment:
no code comments! based on our 1on1, you'll want to potentially either remove this or add numerically allow_blank: true back in